festim/generic_simulation.py

```diff
@@ -363,7 +363,7 @@ def run_post_processing(self):
         self.update_post_processing_solutions()
 
         self.exports.t = self.t
-        self.exports.write(self.label_to_function, self.dt)
+        self.exports.write(self.label_to_function, self.dt, self.mesh.dx)
```

Comment:
Should `mesh.dx` be an attribute of `exports` instead?

Reply:
This might be better as we might need a way to check in the TrapDensityXDMF the number of subdomains to make the variational formulation properly

Reply:
I don't see how having the dx measurement an attribute of exports vs an argument of exports.write makes you more able to check the number of subdomains... this suggestion was more to refactor the code